### PR TITLE
Setup/react router

### PR DESCRIPTION
**Pull request by philawsophizing**
Cheers @esraajb , happy for any input on this, seems to work well. On to testing.

### CODE REVIEW

**Review by Karyum — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>PAS</title>
+    <base href="/" />
```
Comment (jbesraa) [inline public/index.html]: Why did you add this?

File: src/app.jsx [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import Counter from './components/counter.jsx';
 
-const App = () => <Counter />;
+const App = () =>
```
Comment (jbesraa) [inline src/app.jsx]: Why do you need this file ?

File: src/routes.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, browserHistory } from 'react-router-dom';
+import App from './app.jsx';
+
+const Routes = () =>
+  (
+    <div>
+      <Route exact path='/' component={App} history={browserHistory} />
```
Comment (jbesraa) [inline src/routes.jsx]: You can require Counter in here and then it will look like that: 
<Route exact path="/whatever" component={counter} />

File: src/routes.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+      <Route path='/*' component={App} history={browserHistory} />
```
Comment (jbesraa) [inline src/routes.jsx]: General Note: you are using react-router v4, its new. almost 90% of whats online is for <v4. pay attention to that.

**Comment by Karyum** [comment]
@esraajb added blocked because its been 6 hours since this pull request was up 
any reason why you merged the latest one and not this one ?? :maple_leaf:

**Review by jbesraa — CHANGES_REQUESTED** [review]

**Comment by philawsophizing** [comment]
@esraajb changes as requested. Thanks :maple_leaf:
